holmes/core/conversations_worker/realtime_manager.py

```
@@ -465,7 +386,56 @@ def _channel_unhealthy(self) -> Optional[str]:
+            # auto-reconnect is unreliable on clean closes, so we do our own
+            # full teardown/reconnect on any failure signal.
+            unhealthy_reason = self._channel_unhealthy()
+            if unhealthy_reason is not None:
+                logging.warning(
+                    "Realtime channel unhealthy (%s), reconnecting",
+                    unhealthy_reason,
+                )
+                self._connected = False
+                try:
+                    self._wake_all()
+                except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)